.github/ISSUE_TEMPLATE/improve-the-site.yml

```diff
@@ -1,30 +1,31 @@
 name: Improve the docs.github.com site
 description: Make a suggestion or report a problem about the technical implementation of docs.github.com.
+title: "'[What I really want the title to be]': "
```

Comment:
Is this supposed to be here or was it a placeholder? It's in all three files.

Reply:
Ack! This must have come back in while resolving merge conflicts. Deleting them now!